it/v1/endpoints/AuthMainAgentISpec.scala

```diff
@@ -22,10 +22,10 @@ import play.api.http.Status
 import play.api.libs.json.{JsValue, Json}
```

Comment:
Can you remove this class AuthMainAgentISpec.scala from v1.endpoints as it already exists in auth package.